src/main/java/org/osiam/addons/self_administration/template/TemplateConfiguration.java

```diff
@@ -10,7 +10,7 @@
 public class TemplateConfiguration {
 
     @Bean
-    public ClassLoaderTemplateResolver emailTemplateResolver() {
+    public ClassLoaderTemplateResolver defaultTemplateResolver() {
```

Comment:
because spring boot creates it's own one [if no `defaultTemplateResolver` bean found](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/thymeleaf/ThymeleafAutoConfiguration.java#L67)... But then by default spring checks the default location of the templates and I need to disable the check with https://gist.github.com/dacrome/0f9226cd859758c877a3